tslint.json

```diff
@@ -37,8 +37,7 @@
     "quotemark": [
       true,
       "double",
-      "avoid-escape",
-      "avoid-template"
+      "avoid-escape"
```

Comment:
1 rule is not known by tslint-to-eslint-config to have an ESLint equivalent:
  * tslint-to-eslint-config does not know the ESLint equivalent for TSLint's "completed-docs".